nexx_integration.module

```diff
@@ -16,10 +16,10 @@ use Drupal\media_entity\Entity\Media;
 function nexx_integration_taxonomy_term_insert(EntityInterface $term) {
   $response_data = nexx_integration_taxonomy_term_request($term, 'insert');
 
-  if (!empty($response_data['item'])) {
+  if (!empty($response_data['result'])) {
     $drupal_id = $term->id();
-    $nexx_id = $response_data['item'];
-    $vid = $term->bundle();
+    $nexx_id = $response_data['result']['itemupdate']['generatedID'];
+    $vid = $term->get('vid')->getString();
```

Comment:
Why has been `$vid = $term->bundle();` changed to `$term->get('vid')->getString();`?